### PR TITLE
:key: Disambiguation for helpviewer keywords in WPF, Winforms, VB

### PR DESCRIPTION
**Pull request by mikeblome**
This is a redo of two earlier PRs only this time no reformatting of tags.

### CODE REVIEW

File: docs/framework/winforms/advanced/about-gdi-managed-code.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ ms.topic: "article"
 helpviewer_keywords: 
   - "GDI+, about GDI+"
```
Comment (mairaw) [inline docs/framework/winforms/advanced/about-gdi-managed-code.md]: not related to your PR and not asking you to fix this, but having an entry GDI+ and another one GDI+, about GDI+ is not super helpful IMO. Thoughts?

Comment (mikeblome) [reply docs/framework/winforms/advanced/about-gdi-managed-code.md]: Agreed. I saw many things like this that I would fix if I had a few free days :-) Some topics have way too many keywords, some too few, and some keywords aren't really useful.

Comment (mairaw) [reply docs/framework/winforms/advanced/about-gdi-managed-code.md]: Cool. We'll address this separately. Don't worry about this. You're already helping a lot @mikeblome!

**Review by mairaw — APPROVED** [review]
LGTM after fixing space issues. I'll merge once it finishes building.

**Pull request merged**